packages/assets-controllers/src/index.ts

```diff
@@ -13,7 +13,30 @@ export type {
   NftMetadata,
 } from './NftController';
 export { getDefaultNftControllerState, NftController } from './NftController';
-export * from './NftDetectionController';
+export type {
```

Comment:
To review this, look through `NftDetectionController.ts` and make sure that we are listing all exports there minus `AllowedActions` and `AllowedEvents`.